partner_firstname/static/description/index.html

```diff
@@ -466,6 +467,10 @@ <h2><a class="toc-backref" href="#toc-entry-7">Contributors</a></h2>
 <li>Graeme Gellatly &lt;<a class="reference external" href="mailto:graeme&#64;o4sb.com">graeme&#64;o4sb.com</a>&gt;</li>
 <li>Laurent Mignon &lt;<a class="reference external" href="mailto:laurent.mignon&#64;acsone.eu">laurent.mignon&#64;acsone.eu</a>&gt;</li>
 <li>Bjorn Billen &lt;<a class="reference external" href="mailto:bjorn.billen&#64;dynapps.be">bjorn.billen&#64;dynapps.be</a>&gt;</li>
+<li><a class="reference external" href="https://aiontech.company/">Aion Tech</a>:<ul>
```

Comment:
Same here, let's leave automatic changes to the bot.

Reply:
This is an automatic change done by **pre-commit** installed locally when I commit, since it ensures coding conventions I'd rather leave it installed; do I really have to change this in order to have the PR merged?

Reply:
@SirAionTech that's more for func ppl :wink: 
As a techie I would recommend setting this env var: `SKIP=oca-gen-addon-readme`